consensus/src/processes/traversal_manager.rs

```diff
@@ -85,13 +105,13 @@ impl<T: GhostdagStoreReader, U: ReachabilityStoreReader, V: RelationsStoreReader
                     "[TRAVERSAL MANAGER] Traversal count: {}, queue size: {}, anticone size: {}, visited size: {}",
                     traversal_count,
                     queue.len(),
-                    anticone.len(),
+                    output.len(),
                     visited.len()
                 );
             }
 
-            if !self.reachability_service.is_dag_ancestor_of(block, current) {
-                anticone.push(current);
+            if !return_anticone_only || !self.reachability_service.is_dag_ancestor_of(block, current) {
```

Comment:
For code clarity, can we do:
```
let is_block_anticone_of_current = !self.reachability_service.is_dag_ancestor_of(block, current);

if !return_anticone_only || is_block_anticone_of_current {
  output.push(current);
}
```

It'd make it easier for whoever is reading this to understand the intent with self-documenting code.